src/components/home/HomeList.js

```diff
@@ -0,0 +1,50 @@
+import { React, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import HomeItem from './HomeItem';
+import { getDrivers } from '../../redux/drivers/DriversSlice';
+import f1Logo from '../../assets/images/f1Icon.png';
+
+const HomeList = () => {
+  const dispatch = useDispatch();
+  let { driverItems } = useSelector((store) => store.drivers);
+  useEffect(() => {
+    if (driverItems.length === 0) {
+      dispatch(getDrivers());
+    }
+  });
+
+  const sumTot = driverItems[driverItems.length - 1];
+  driverItems = driverItems.slice(0, -1);
+
+  return (
+    <div className="home_lowerContainer">
+      <div className="home_topGeneral">
+        <img alt="formula 1 logo" src={f1Logo} />
+        <div id="home_sum">
+          <span id="home_title">FORMULA 1</span>
+          <div>
+            <span>{sumTot}</span>
+            <span> drivers</span>
+          </div>
+        </div>
+      </div>
+      <div id="home_statsContainer">
+        <span id="home_stats">STATS BY CONSTRUCTOR</span>
+      </div>
+      <div className="home_listItems">
+        {driverItems.map((driver) => (
+          <HomeItem
+            key={driver.id}
+            driverId={driver.id}
+            constructorImg={driver.constructor_img}
+            constructorName={driver.constructor_name.replace('_', ' ')}
+            numDrivers={driver.num_drivers}
+            contIds={driverItems.indexOf(driver) + 1}
+          />
+        ))}
+      </div>
```

Comment:
- [ ] 👏 Great job so far in building your capstone project! 🚀

    - While your work has been impressive, I'd like to bring your attention to a specific requirement that appears to be missing from the project's specifications.

    - As outlined in the project requirements, there's a crucial element yet to be implemented: the `filter` functionality. This feature is designed to assist users in swiftly sorting through the lists of categories on the home page, focusing particularly on the category of "cars" in your context. Integrating this `filter` functionality holds significance as it enhances user experience by allowing them to save time, avoiding the need to manually scroll through the page in search of a specific car category. Instead, users can simply enter their desired query into the search bar and quickly locate what they're looking for. 🚗💨 See below for more details on project requirements:
    
<img width="1010" alt="Screenshot 2023-09-07 at 19 04 16" src="https://github.com/ivangonzalez224/F1-Drivers-Metrics/assets/107192076/f8777b46-a770-44aa-b4df-98c0d14ca8b8">


Please consider prioritizing this feature to ensure your project aligns with the defined requirements and provides a smoother experience for your users. Keep up the excellent work! 🌟👍

Reply:
Hi @Strangeal thank you for your feedback.
I already implemented the filter functionality you suggested, by adding a search bar in the home page that allows the user to filter details by entering the constructor name in order to enhance the user experience.